src/integration-tests/commonTests.js

```diff
@@ -2,30 +2,36 @@ var supertest = require('supertest') //require supertest
 var defaults = require('superagent-defaults')
 const crypto = require('crypto')
 let baseURL = 'http://localhost:6001'
+const waitOn = require('wait-on')
 //supertest hits the HTTP server (your app)
 let defaultHeaders
 
+// Utility function to wait for services to be ready
+const waitForService = async (url) => {
+	const opts = {
+		resources: [url],
+		delay: 5000,
+		interval: 2500,
+		timeout: 100000,
+	}
+	await waitOn(opts)
+}
+
 const logIn = async () => {
 	try {
 		console.log('============>LOGIN 1 : ')
```

Comment:
If we need console.log for any specific debugging, can we provide some meaningful statements inside? For example instead of 
============>LOGIN 1 :
we can have , 
============> ATTEMPTING LOGIN  :
and so on... 
@priyanka-TL